01.dqn.ipynb

```diff
@@ -21,7 +21,7 @@
     "    !apt install ffmpeg\n",
     "    !apt install xvfb\n",
     "    !pip install PyVirtualDisplay==3.0\n",
-    "    !pip install gymnasium == 0.28.1\n",
+    "    !pip install gymnasium\n",
```

Comment:
Can fix the error by removing the spaces (gymnasium==0.28.1), or just removing the version specifier.